dev-docs/bidders/freedomadnetwork.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: bidder
+title: Freedom Ad Network
+description: Freedom Ad Network Bidder Adapter
+biddercode: freedomadnetwork
+aliasCode: fan
+gvl_id: none
+coppa_supported: true
+media_types: banner, native
+safeframes_ok: false
+deals_supported: false
+floors_supported: false
+fpd_supported: true/false
+pbjs: true
+prebid_member: false
```

Comment:
Please provide as much meta data as possible
https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

```
tcfeu_supported: true/false
dsa_supported: true/false
gvl_id: none
usp_supported: true/false
coppa_supported: true/false
gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp
schain_supported: true/false
dchain_supported: true/false
userId: (list of supported vendors)
media_types: banner, video, native
safeframes_ok: true/false
deals_supported: true/false
floors_supported: true/false
fpd_supported: true/false
pbjs: true/false
pbs: true/false
prebid_member: true/false
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
ortb_blocking_supported: true/partial/false
privacy_sandbox: no or comma separated list of `paapi`, `topics`
```

Missing values result in `check with bidder` being rendered, which makes vetting your adapter harder